packages/types/src/types.ts

```diff
@@ -95,6 +95,11 @@ export interface Bookmark {
     createdAt: string;
 }
 
+export interface UpdateBookmarksPayload {
+    bookmarkKey: string;
```

Comment:
If I understand it correctly, `bookmarkKey` is used to determine whether a bookmark has been added or deleted. If it is in the payload it has been added, if not it has been deleted. Is my understanding correct?
For synchronization this action is also used to update the `bookmarks` property of the state, without a need for `bookmarkKey`. Can we make this optional?